src/screens/AddSiteFormScreen.js

```diff
@@ -335,67 +337,75 @@ class AddSiteFormScreen extends Component {
                         {site_info}
                     </Text>
 
-                    <FormLabel>{title}</FormLabel>
-                    <FormInput
+                    <Text style={labelStyle}>{title}</Text>
+                    <Input
+                        autoCapitalize={'words'}
+                        containerStyle={formInputStyle}
                         placeholder={add_site_title}
                         value={siteTitleText}
                         onChangeText={this.onUpdateSiteTitleText}
                     />
 
-                    <FormLabel>{description}</FormLabel>
-                    <FormInput
+                    <Text style={labelStyle}>{description}</Text>
+                    <Input
+                        containerStyle={formInputStyle}
                         placeholder={description_placeholder}
                         value={siteDescriptionText}
                         onChangeText={this.onUpdateSiteDescriptionText}
                         containerStyle={largeTextInput}
+                        blurOnSubmit={true}
                         multiline={true}
-                        maxHeight={50}
+                        autoGrow={true}
```

Comment:
Some nice helpers here... `blurOnSubmit` lets us use the `Return` key in the keyboard to exit the multi-line input instead of it just making a new line. `autoGrow` is in RN 0.54 which we aren't on yet, but when Expo's SDK 26 comes out they'll be moving to that RN version, and we'll get this fix for free.